OTKit/otkit-typography-desktop/token.yml

```diff
@@ -34,6 +34,19 @@ props:
     type: "raw"
     value: "'BrandonText', 'Helvetica Neue', Helvetica, Arial, OpenTableHiragino, Roboto, Droid Sans, '游ゴシック体', OpenTableYuGothic, YuGothic, Yu Gothic, 'メイリオ', Meiryo, 'ＭＳ Ｐゴシック', 'MS PGothic', sans-serif"
 
+  # Design decisions for font group xxxsmall-regular.
+  #
+  # =============================================
+  xxxsmall-regular-font-size:
```

Comment:
`xsmall` font size is `12px`. Maybe consider this `xxsmall` instead of `xxx`. 

xsmall:
https://github.com/opentable/design-tokens/blob/77b331c29c34abcbb6e27a00c33508f1ead08f8e/OTKit/otkit-typography-desktop/token.yml#L41

Reply:
xsmall is 14px, and xxsmall is 12 px in this file

Reply:
HA. well clearly i've forgotten how to read 😓 . Ignore my comment